### DISCUSSION TITLE
PolymorphicChildModelAdmin list displays base model of foreignkey fields

### DISCUSSION THREAD

**Discussion by variable** [jazzband/django-polymorphic#805] [Q&A] [answered]
I am aware of the `polymorphic_list` field in the `PolymorphicParentModelAdmin`, but I am using `PolymorphicChildModelAdmin`, and I am wondering if it's possible to display the relationship field to it's child class.

e.g

<pre><code>
class Vote(Model):
  #  item can be "Content", "User" which extends "Trackable"
  item = Trackable(..)
</code></pre>

When I display the admin list, the item is always displayed as "Trackable"

**Comment by vdboor** [comment]
I'm not sure I get this.. You have a `VoteAdmin` or `TrackableAdmin` child? could you post a more complete example? the code you show clearly isn't Django model code, I'd expect it to be:

```
class Vote(models.Model):
    item = models.ForeignKey(Trackable)
```

**Comment by JohananOppongAmoateng** [comment]
@bckohan I think this should be a discussion

**Comment by JohananOppongAmoateng** [comment, marked as answer]
@variable 
`polymorphic_list` only works on `PolymorphicParentModelAdmin` because it relies on a polymorphic queryset to resolve and display fields from different child models.

`PolymorphicChildModelAdmin` is already scoped to a single concrete child model and uses a non-polymorphic queryset, so there’s no equivalent mechanism to automatically display relationship fields that belong to *other* child classes.

You can display:

* fields defined on the child model itself, and
* fields defined on the parent model.

If you need to show data shared across multiple children, the recommended approach is to move that relationship to the parent model. Otherwise, the only workaround is a custom `list_display` method that manually accesses related objects, which is read-only and must be handled carefully.

This limitation is intentional and consistent with Django admin’s model isolation, not a missing feature in django-polymorphic.
